adafruit_si7021/__init__.py

```diff
@@ -137,6 +138,7 @@ class SI7021:
     def __init__(self, i2c_bus: I2C, address: int = 0x40) -> None:
         self.i2c_device = I2CDevice(i2c_bus, address)
         self._command(_RESET)
+        time.sleep(0.015)
```

Comment:
Do you think adding a comment here to explain why there's a use of `time.sleep()` would be helpful?

Reply:
Yep, definitely. Latest comment adds a note with link back to the datasheet.